conf/messages.cy

```diff
@@ -397,7 +397,7 @@ submitted.reference = Cyfeirnod ECL: {0}
 submitted.subHeading = Yr hyn sy’n digwydd nesaf
 submitted.p1 = Rydych wedi cofrestru’ch cymhwystra ar gyfer yr Ardoll Troseddau Economaidd (ECL). Rydych wedi cael cyfeirnod ac e-bost cadarnhau. Cadwch y rhain ar gyfer eich cofnodion.
 submitted.p2 = Bydd angen i chi gyflwyno datganiad a thalu’r ardoll yn flynyddol.
-submitted.p3 = Y dyddiad cau nesaf ar gyfer cyflwyno datganiad a gwneud taliad yw 30 Medi 2023. Codir llog arnoch os ydych yn talu’n hwyr.
+submitted.p3 = Y dyddiad cau nesaf ar gyfer cyflwyno datganiad a gwneud taliad yw {0}. Codir llog arnoch os ydych yn talu’n hwyr.
```

Comment:
Does the code already pass the date when using this message?

Reply:
Yeah @paddynski-opencast, the English version works fine, and with the change, the Welsh does as well.